src/worker_pool/index.ts

```diff
@@ -167,6 +95,10 @@ export class WorkerInfo extends AsynchronouslyCreatedResource {
     }
 
     _handleResponse (message : ResponseMessage) : void {
+      if (message.time != null) {
+        this.histogram?.record(Math.max(1, message.time));
```

Comment:
I do not understand the Math.max: if the time-stamping can give a zero execution time, there's an issue with it.

Reply:
I noticed this from a previous issue, where the problem was non-deterministic, couldn't reproduce it easily.

I added it as a workaround but need to dig deeper into it

Reply:
With the high resolution timestamping at ns?

Reply:
Yeah, got issue reports on that and I also was able to reproduce it from time to time, but was not deterministic. Every N times could reproduce it.

In theory, when having `0` out of the high ms resolution with `performance.now`, means start of the process, so I imagine goes in that sense.

Reply:
Sounds like a bug to me. But dunno where it came from.